### DISCUSSION TITLE
Adding EasyDNS to the App

### DISCUSSION THREAD

**Discussion by Starlord0006** [qdm12/ddns-updater#350] [Ideas]
Hi,
     Is there any plans to add EasyDNS to DDns provider list

     https://easydns.com
     https://kb.easydns.com/knowledge/dynamic-dns/

     All explain in this page.

Thanks

**Comment by qdm12** [comment]
Done in 6a9ba7ecb42c94ff0cdc4051139997b34b1db672 for the latest image and future release image tag `:v2.6.0`
